src/modules/35-connectors/modals/ConnectorModal/__mocks__/useCreateConnectorModal.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { useState } from 'react'
+import { Button } from '@wings-software/uicore'
+import { useModalHook } from '@harness/use-modal'
+import { Dialog, IDialogProps } from '@blueprintjs/core'
+
+let mockData: any = null
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9238322)
```suggestion
let mockData: unknown = null
```